src/main/kotlin/lotto/controller/TicketsEvaluator.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.controller
+
+import lotto.domain.Rank
+import lotto.domain.WinningCombination
+import lotto.dto.IssuedTickets
+
+object TicketsEvaluator {
+    fun runWith(issuedTickets: IssuedTickets, winningCombination: WinningCombination): List<Rank> {
+        return issuedTickets.get().map {
+            it.getRank(winningCombination)
+        }
+    }
```

Comment:
I think the best to practice is how View, Controller, and Domain interact with each other,
but I still think this kind of attempt is valuable.

As you try out different approaches like this, I believe you'll naturally start thinking about better structures.

For now, I understand what you were aiming for, so be great to refine this a bit more!